apps/site/components/Containers/NavBar/index.tsx

```diff
@@ -53,17 +53,21 @@ const NavBar: FC<NavbarProps> = ({
         <Link className={style.nodeIconWrapper} href="/" aria-label="Home">
           <WithNodejsLogo />
         </Link>
-
         <Label.Root
-          onClick={() => setIsMenuOpen(prev => !prev)}
           className={style.sidebarItemTogglerLabel}
           htmlFor="sidebarItemToggler"
         >
           {navInteractionIcons[isMenuOpen ? 'close' : 'show']}
         </Label.Root>
       </div>
 
-      <input className="peer hidden" id="sidebarItemToggler" type="checkbox" />
+      <input
+        className={`peer ${style.sidebarItemToggler}`}
+        id="sidebarItemToggler"
```

Comment:
Is an ID required if there is no label referencing the ID?

Reply:
There is a label, defined on line 56
https://github.com/nodejs/nodejs.org/blob/39d57a68a67841252a17279ee64804967b52e8c4/apps/site/components/Containers/NavBar/index.tsx#L56-L59

Reply:
Silly me!